Userland/Applications/Piano/RollWidget.cpp

```diff
@@ -185,61 +185,63 @@ void RollWidget::mousedown_event(GUI::MouseEvent& event)
     if (!widget_inner_rect().contains(event.x(), event.y()))
```

Comment:
As far as I can tell this last commit can be a separate PR, since it fixes something entirely unrelated to the first two commits.